### ISSUE TITLE
How do I change the spellchecker language when Emacs is running?

### ISSUE THREAD

**Issue by kuba-orlik** [hlissner/doom-emacs#3942]
When using the `spell` module, how do I change which dictionary should be used for a given buffer?

I've opened a text file, wrote some words in Polish and they all got a red underline. Then I used `ispell-change-dictionary` and chose `polish`, but nothing changed - the words, even though correctly spelled, are still underlined, unless they are proper *English* words.

How do I effectively change the dictionary?

**Comment by ymarco** [comment]
The normal package (`spell-fu`) supports only English as [far as I know](https://gitlab.com/ideasman42/emacs-spell-fu/-/issues/9), but the old backend `flyspell` (which can be enabled with a `+flyspell` flag on the `spell` module) does support this.

tldr: use `(spell +flyspell)`

**Comment by kuba-orlik** [comment]
That worked! Thanks

**Comment by n14s** [comment]
No proszę!
Kuba, I'm having the same issue, but using "(spell +flyspell)" didn't solve it.
Activating spell-fu results in red lines underneath every non-english word (with German ispell dict active). Activating flyspell-mode results in no red lines at all. 
Can you help me out?

**Comment by kuba-orlik** [comment]
Jasne! :)

In order to change the language, I first run `M-x ispell-change-dictionary`, and then enable `flyspell` mode.

I think it'd be nice if the `spell` module had a README like most other modules. I'll try to contribute one once I free some time to do it (also won't mind if anyone manage to do it first :) )
